### PR TITLE
Post install script second atttempt

### PR DESCRIPTION
**Pull request by navihtot**

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -169,6 +171,32 @@ Lambda.prototype._npmInstall = function (program, codeDirectory, callback) {
   });
 };
 
+Lambda.prototype._postInstallScript = function (codeDirectory, callback) {
+  var script_filename='post_install.sh';
```
Comment (DeviaVir) [inline lib/main.js]: Please use `var script_filename = 'post_install.sh';`

File: lib/main.js [hunk 2]
```diff
@@ -100,6 +100,8 @@ Lambda.prototype._runHandler = function (handler, event, runtime) {
 
 };
 
+
+
```
Comment (DeviaVir) [inline lib/main.js]: Why the extra newlines here?

**Comment by DeviaVir** [comment]
Thanks for this @navihtot! I'll merge as soon as the two nitty comments have been addressed!

**Comment by navihtot** [comment]
There :)

**Comment by DeviaVir** [comment]
:+1:!

**Pull request merged**